scripts/functions/gemset

```diff
@@ -355,7 +355,7 @@ gemset_initial()
 run_gem_wrappers()
 {
   gem_install gem-wrappers >/dev/null &&
-  gem pristine gem-wrappers > /dev/null &&
+  ([ "$(printf '%s\n' "3.2.3" "$(gem -v)" | sort -V | head -n1)" == "3.2.3" ] && gem pristine gem-wrappers --only-plugins > /dev/null || exit 0) &&
```

Comment:
If you're going by rubygems version, maybe "3.2.0" would be a better choice. It looks like that's when the relevant change happened.

Reply:
@BrianHawley Great suggestion! The `only-plugins` option is available since 3.2.0 🎉 

https://github.com/rubygems/rubygems/blob/v3.2.0/lib/rubygems/commands/pristine_command.rb#L42-L45

Reply:
In rubygems < 3.2.0 (embedded in rubies 2.x), the options `only-plugins` does not exists and not need "regenerate and pristine" the wrappers. But, in ruby 3 (that embeds inside itself a rubygems >= 3.2.0), it is needed regenerate because of some internal changes in rubygems, and also needs a condition in shells of rvm codebase. Now it is done! 🏁